src/saveToCase.js

```diff
@@ -568,6 +687,25 @@ define([
                         return;
                     }
                 }
+                
+                var attachmentRegex = /\/case\/attachment\/(\w+)\/@src/,
```

Comment:
Is this expected to match at the end of the string only? If yes, you probably want to add `$`.